### PR TITLE
ENCD-3981 remove histone modification

### PR DESCRIPTION
**Pull request by jahilton**

### CODE REVIEW

File: src/encoded/tests/test_upgrade_target.py [hunk 1]
```diff
@@ -70,3 +81,10 @@ def test_target_upgrade_status_5_6(upgrader, target_5):
         'target', target_5, current_version='5', target_version='6')
     assert value['schema_version'] == '6'
     assert value['status'] == 'current'
+
+
+def test_target_upgrade_status_6_7(upgrader, target_6):
```
Comment (keenangraham) [inline src/encoded/tests/test_upgrade_target.py]: You could rename this function to something like `test_target_upgrade_remove_histone_modification_6_7`.

File: src/encoded/tests/test_upgrade_target.py [hunk 2]
```diff
@@ -38,6 +38,17 @@ def target_5(target):
     return item
 
 
+@pytest.fixture
+def target_6(target):
+    item = target.copy()
+    item.update({
+        'schema_version': '5',
```
Comment (keenangraham) [inline src/encoded/tests/test_upgrade_target.py]: I think this should be schema_version six.

File: src/encoded/tests/test_upgrade_target.py [hunk 3]
```diff
@@ -38,6 +38,17 @@ def target_5(target):
...
+        'status': 'proposed',
```
Comment (keenangraham) [inline src/encoded/tests/test_upgrade_target.py]: Proposed no longer exists.

File: src/encoded/tests/test_upgrade_target.py (hunk 1 above)
Comment (jahilton) [reply src/encoded/tests/test_upgrade_target.py]: renamed the function & pushed change

File: src/encoded/tests/test_upgrade_target.py (hunk 2 above)
Comment (jahilton) [reply src/encoded/tests/test_upgrade_target.py]: schema_version:6 and pushed change

File: src/encoded/tests/test_upgrade_target.py (hunk 3 above)
Comment (jahilton) [reply src/encoded/tests/test_upgrade_target.py]: status:current and pushed change

**Pull request merged**